python/vehicle-routing/src/vehicle_routing/score_analysis.py

```diff
@@ -41,3 +42,14 @@ def __post_init__(self):
         self.description = (f"Vehicle '{self.vehicle_name}' total travel time is "
                             f"{self.total_driving_time_seconds // (60 * 60)} hours "
                             f"{round(self.total_driving_time_seconds / 60)} minutes.")
+
+
+@dataclass
+class ServiceFinishedAfterMaxEndTimeJustification(ConstraintJustification):
```

Comment:
We've agreed a while back that we won't be adding justifications to the quickstarts.
Please remove; if the Java quickstart still has them, that's an accident and please remove those too.